frontend/src/pages/ServicesPage.jsx

```diff
@@ -0,0 +1,136 @@
+import React, { useEffect, useState } from "react";
+import Header from "../Components/Header/Header";
+import Main from "../Components/Main/Main";
+import { Row, Tabs, Table, Button } from "antd";
+import { AiOutlinePlus } from "react-icons/ai";
+import { useParams } from "react-router-dom";
+import api from "../lib/api";
+import useNotification from "../util/useNotification";
+import { useCookies } from "react-cookie";
+import store from "../lib/store";
+
+const ServicesPage = () => {
+  const columns = [
+    {
+      title: "Sort by",
+      dataIndex: "name",
+    },
+    {
+      title: "Submittant",
+      dataIndex: "submitter",
+    },
+    {
+      title: "Attachments",
+      dataIndex: "attachments",
+    },
+    {
+      title: "Submittance data",
+      dataIndex: "submitted_date",
+    },
+    {
+      title: "Action",
+      dataIndex: "",
+      render: () => {
+        return (
+          <Button className="add-btn" type="primary">
+            <AiOutlinePlus />
+          </Button>
+        );
+      },
+    },
+  ];
+
+  const panes = [
+    {
+      title: "Pending",
+      key: "1",
+    },
+    {
+      title: "Approved",
+      content: "Approved",
+      key: "2",
+    },
+    {
+      title: "Rejected",
+      key: "3",
+    },
+  ];
+
+  const { id } = useParams();
+  const { notify } = useNotification();
+  const [cookies] = useCookies(["AUTH_TOKEN"]);
+  const { dashboardData } = store.currentState;
+  const [data, setData] = useState(null);
+  const [selectTab, setSelectTab] = useState({
+    activeKey: panes[0].key,
+    panes,
+  });
+  const [status, setStatus] = useState("pending");
+
+  const handleTabsChange = (activeKey) => {
+    const activePane = panes.find((p) => p.key === activeKey);
+    setSelectTab({ activeKey });
+    setStatus(activePane?.title.toLowerCase());
+  };
+
+  useEffect(() => {
+    api
+      .get(`/data?form_id=${id}&status=${status}&page=1&perpage=10`, {
+        headers: {
```

Comment:
you don't need a header when you already have a token added to the `api` using `api.setToken(thetoken)`